### PR TITLE
Unittest ag third party

### PR DESCRIPTION
**Pull request by sjanssen2**

### CODE REVIEW

**Comment by wasade** [comment]
:+1:

File: knimin/tests/test_ag_third_party.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+        response = self.get('/ag_third_party/add/')
+        self.assertEqual(response.code, 200)
+        form = NewThirdParty()
+        # test that all fields are present in the HTML side.
+        for key, element in form.__dict__['_fields'].items():
+            self.assertIn(str(element.label), response.body)
+
+    def test_post(self):
+        self.mock_login_admin()
+        # form = NewThirdParty()
```
Comment (mortonjt) [inline knimin/tests/test_ag_third_party.py]: Remove?

**Comment by mortonjt** [comment]
I think it is good to merge once that comment is removed.

**Comment by wasade** [comment]
agree with @mortonjt

**Pull request merged**